tests/baselines/reference/privateNameNotAccessibleOutsideDefiningClass.errors.txt

```diff
@@ -0,0 +1,17 @@
+tests/cases/conformance/classes/members/privateNames/privateNameNotAccessibleOutsideDefiningClass.ts(7,9): error TS2339: Property '#foo' does not exist on type 'A'.
+tests/cases/conformance/classes/members/privateNames/privateNameNotAccessibleOutsideDefiningClass.ts(7,9): error TS18013: Property '#foo' is not accessible outside class 'A' because it has a private name.
+
+
+==== tests/cases/conformance/classes/members/privateNames/privateNameNotAccessibleOutsideDefiningClass.ts (2 errors) ====
+    // @target es6
+    
+    class A {
+        #foo: number = 3;
+    }
+    
+    new A().#foo = 4;               // Error
+            ~~~~
+!!! error TS2339: Property '#foo' does not exist on type 'A'.

```

Comment:
shouldn't give two errors. the first one is the wrong one, I think.

Reply:
resolving because we're discussing  this elsewhere